files/tasks/rpm-test-deps.yaml

```diff
@@ -12,10 +12,11 @@
     state: present
   become: true
 # flexmock is not in EPEL 9, install it from PyPI
-- name: Install flexmock from PyPI
+- name: Install flexmock and pyforgejo from PyPI
   pip:
     name:
       - flexmock
+      - pyforgejo
```

Comment:
Technically speaking it's not a test dependency, shouldn't the one line in `zuul-install-requirements-pip.yaml` be enough?